frame/support/procedural/src/pallet/parse/mod.rs

```diff
@@ -62,6 +66,25 @@ pub struct Def {
 }
 
 impl Def {
+	fn mark_benchmarking_cached_calls(storage_def: &mut StorageDef, item: &mut syn::Item) {
+		if let syn::Item::Type(typ) = item {
+			for attr in typ.attrs.as_slice() {
```

Comment:
This is basically:

```suggestion
			for attr in &typ.attrs {
```

?